pkg/pillar/hypervisor/kvm.go

```diff
@@ -1163,26 +1163,26 @@ func waitForQmp(domainName string, available bool) error {
 	delay := time.Second
 	var waited time.Duration
 	var err error
+	sock := GetQmpExecutorSocket(domainName)
 
 	for {
+		if waited > maxDelay {
```

Comment:
I was wondering the same. But currently this waits for a max of 10 seconds, so unless getQemuStatus() fails immediately (not due to a timeout) there will be no wait in here.
Note that 4*15 seconds would be a lot since a watchdog might fire after 300 seconds - depends on the code path which might be calling this.

But if getQemuStatus() always sleeps and retries on all errors, then I think we can remove this loop.

Reply:
@rene I removed this (and added comments)